2023/ego-and-math/german/sentence_translations.json

```diff
@@ -172,7 +172,7 @@
  },
  {
   "input": "I don't think it was the intention of any of the adults around. ",
-  "translatedText": "Ich glaube nicht, dass das die Absicht eines der Erwachsenen war. ",
+  "translatedText": "Ich glaube auch nicht, dass das überhaupt von den Erwachsenen beabsichtigt war.",
```

Comment:
I think this i missing the 'any of the adults' part because i think here he brings in the fact that other adults, apart from the teachers, also didn't want it to feel competitive. Maybe something like '..., dass das überhaupt von irgendeinem Erwachsenen beabsichtigt war'. The überhaupt is is nice addition!